tests/entrypoints/test_chat_utils.py

```diff
@@ -264,10 +264,8 @@ def test_parse_chat_messages_multiple_images(
                     "url": image_url
                 }
             }, {
-                "type": "image_url",
-                "image_url": {
-                    "url": image_url
-                }
+                "type": "image",
```

Comment:
I prefer to have
```suggestion
                "type": "image_pil",
```

given that we add `image_embed`